web/app/components/comment/comment.jsx

```diff
@@ -244,7 +244,7 @@ export default class Comment extends Component {
 
     if (mods.view === 'preview') {
       return (
-        <div className={b('comment', props, defaultMods)}>
+        <div className={b('comment', props, defaultMods)} role="listitem article" aria-level={mods.level}>
```

Comment:
Instead of `role="article"` I think you should change `div` with `article`.

But don't forget to add `display: block` to `.comment` in this case.